### DISCUSSION TITLE
Other Upscalers

### DISCUSSION THREAD

**Discussion by RandomKaos999** [painebenjamin/app.enfugue.ai#123] [General]
Hi, me again.

I was wondering if it was possible to add different upscaler models for usage?
I attempt to place 4xUltraSharp.pth into .cache\enfugue\other with the other upscaling models that I found, but it doesn't appear to be added to the list. Is there more to it? or currently not possible?

Thanks!

**Comment by painebenjamin** [comment]
Hello again!

At the present moment, it's not possible; the available upscalers are presently hardcoded. This isn't too hard to support, though, so I've made an issue from this discussion to add it in.

Thanks for the suggestion!
